pom.xml

```diff
@@ -200,7 +200,13 @@
 			<dependency>
 				<groupId>org.mockito</groupId>
 				<artifactId>mockito-core</artifactId>
-				<version>3.2.4</version>
+				<version>4.2.0</version>
+				<scope>test</scope>
+			</dependency>
+			<dependency>
+				<groupId>org.mockito</groupId>
+				<artifactId>mockito-inline</artifactId>
+				<version>3.9.0</version>
```

Comment:
why not use  mockito-inline version 4.2.0 ?

Reply:
Yes we can, but it's also working fine with 3.9.0.

Reply:
> Yes we can, but it's also working fine with 3.9.0.

why not use the current version!!

Reply:
@HerbertYiga done the change. Thanks